torchtext/experimental/datasets/text_classification.py

```diff
@@ -16,6 +16,10 @@ def _build_vocab(data, transforms):
         tok_list.append(transforms(txt))
     return build_vocab_from_iterator(tok_list)
 
+def vocab_func(vocab):
```

Comment:
Not sure why this appears here. But we import `vocab_func` from `torchtext.experimental.functional`

Reply:
Should be fixed now!